test/e2e/storage/vsphere_volume_perf.go

```diff
@@ -69,25 +66,20 @@ var _ = SIGDescribe("vcp-performance [Feature:vsphere]", func() {
 		namespace = f.Namespace.Name
 
 		// Read the environment variables
-		volumeCountStr := os.Getenv("VCP_PERF_VOLUME_COUNT")
-		Expect(volumeCountStr).NotTo(BeEmpty(), "ENV VCP_PERF_VOLUME_COUNT is not set")
-		volumeCount, err = strconv.Atoi(volumeCountStr)
+		Expect(VCPPerfVolumeCount).NotTo(BeEmpty(), "ENV VCP_PERF_VOLUME_COUNT is not set")
```

Comment:
VCPPerfVolumeCount, VCPPerfVolumesPerPod, VCPPerfIterations, SPBMGoldPolicyName, StorageClassDatastoreName not defined.